src/github.com/getlantern/testify/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
We replaced "getlantern/testify" with "stretchr/testify". It should be removed from repo.